devel/libevent/Portfile

```diff
@@ -8,7 +8,7 @@ version             [lindex [split ${version} -] 0]
 categories          devel
 platforms           darwin
 license             BSD
-maintainers         nomaintainer
+maintainers         gmail.com:jerryyhom openmaintainer
```

Comment:
This isn't the correct format any longer for this; you need to provide both your github handle and email address. 

`maintainer {@jerryyhom gmail.com:jerryyhom} openmaintainer`

or something similar.

Reply:
Hm, the MacPorts Guide does not indicate that both are required.  Is there an automatic notification, or can you explain why github handles are required?

Reply:
With the GitHub handle in there, then you will automatically get notified when someone else makes a pull request against your port(s). 

Yes, the guide needs to be updated.

Reply:
Okay, thanks for the explanation.  Probably I am in the minority of maintainers who do not regularly check github.